packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -0,0 +1,176 @@
+import * as setup from "./utilities"
+import { FieldType, Table, ViewV2 } from "@budibase/types"
+import { generator, structures } from "@budibase/backend-core/tests"
+
+function priceTable(): Table {
+  return {
+    name: "table",
+    type: "table",
+    schema: {
+      Price: {
+        type: FieldType.NUMBER,
+        name: "Price",
+        constraints: {},
+      },
+      Category: {
+        type: FieldType.STRING,
+        name: "Category",
+        constraints: {
+          type: "string",
+        },
+      },
+    },
+  }
+}
+
+describe("/views/v2", () => {
+  const request = setup.getRequest()
+  const config = setup.getConfig()
+  let table: Table
+
+  afterAll(setup.afterAll)
+
+  beforeAll(async () => {
+    await config.init()
+    table = await config.createTable(priceTable())
+  })
+
+  const saveView = async (view: ViewV2) => {
```

Comment:
These api functions can be added to the api test clients, so we can use like `config.api.views.foo()`